src/calculateRentalCost.js

```diff
@@ -5,6 +5,22 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const pricePerDay = 40;
+  const longTerm = 7;
+  const longTermDiscount = 50;
+  const shortTerm = 3;
+  const shortTermDiscount = 20;
+  const result = days * pricePerDay;
+
+  if (days >= longTerm) {
+    return result - longTermDiscount;
+  }
+
+  if (days >= shortTerm) {
+    return result - shortTermDiscount;
+  }
```

Comment:
There is a logical issue here. The code currently applies only one discount, either the long-term or short-term, but not both. According to the task requirements, if the number of days is greater than or equal to the long-term threshold, only the long-term discount should be applied. If the number of days is less than the long-term threshold but greater than or equal to the short-term threshold, the short-term discount should be applied. Consider restructuring the conditional logic to reflect this.

Reply:
done